<?xml version="1.0"?>
<launch>
  <node pkg="line_follower" type="line_detector_cnn.py" name="line_detector" clear_params="true" output="screen"> 
    <param name="~with_display" value="0"/>
    <param name="~with_GPU" value="1"/>
    <param name="~mono" value="0"/>
    <param name="~x_width" value="128"/>
    <param name="~x_offset" value="-4"/>
    <param name="~y_height" value="45"/>
    <param name="~y_offset" value="72"/>
    <param name="~model_path" value="/home/pi/catkin_ws/src/line_follower/extras/model_jetson"/>
    <param name="~image_topic" value="/main_camera/image_raw"/>
  </node>
  
</launch>
